src/pages/settings-page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```

Comment:
@AhmedAlamoudi192 please move the file to /routes

Reply:
done